src/util.js

```diff
@@ -55,3 +55,26 @@ exports.extractValueFromObject = (obj, path, def) => {
 
   return current
 }
+
+exports.handleServerError = (ctx, operationFailureMsg, error, logger) => {
```

Comment:
@bradsawadye Why did you move the logger to a param?

Reply:
For some reason, the logger functions do not work when i import logger into the util module. Was getting the error 'logger.error' is not a function

Reply:
this does seem strange and shouldnt be the issue. We dont really want to be passing the logger around where needed. It should just be included in the script making use of it

Reply:
@BMartinos its quite strange but following
`
const logger = require('./logger')

exports.handleServerError = (ctx, operationFailureMsg, error) => {
  ctx.status = ctx.statusCode ? ctx.statusCode : 500
  const err = `${operationFailureMsg}${error.message}`
  ctx.body = {error: err}
  logger.error(err)
}
`
does not work. I get the error logger.error is not a function.

A work round for this that I can do is 
`
let logger = require('./logger')

exports.handleServerError = (ctx, operationFailureMsg, error) => {
if (!logger.error) {
 logger = require('./logger)
}
  ctx.status = ctx.statusCode ? ctx.statusCode : 500
  const err = `${operationFailureMsg}${error.message}`
  ctx.body = {error: err}
  logger.error(err)
}
`
Its not clean though